app/models/ext_management_system.rb

```diff
@@ -542,6 +544,14 @@ def total_vms_never;     vm_count_by_state("never");     end
 
   def total_vms_suspended; vm_count_by_state("suspended"); end
 
+  def supports_block_storage
```

Comment:
@durandom any idea why `alias` would not work here? With `alias supports_block_storage supports_block_storage?`:

```ruby
ems.supports_block_storage? # => true
ems.supports_block_storage # => false
```